tests/integration/arlocal/ar-io-contract/index.js

```diff
@@ -7485,10 +7486,9 @@ var TransferToken = class {
         getInvalidAjvMessage(validateTransferToken, input, 'transferToken'),
```

Comment:
is this stuff supposed to be in here?

Reply:
it's the newest copy of the contract source code that expects everything in mIO